tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -116,6 +116,13 @@ module.exports = {
           .click(declineShareButton)
           .waitForAjaxCallsToStartAndFinish()
       }
+    },
+    getPhoenixcontainer: async function () {
+      let phoenixContainerContains = true
+      await this.api.elements('xpath', '@phoenixContainer', function (result) {
```

Comment:
Dont use locateStrategy like this...
Fetch it from required elements object

Reply:
done